api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -305,6 +305,8 @@ message Bootstrap {
   // field.
   // [#not-implemented-hide:]
   map<string, core.v3.TypedExtensionConfig> certificate_provider_instances = 25;
+
+  google.protobuf.Int64Value max_receive_grpc_message_size = 31;
```

Comment:
@KBaichoo similar to your PR, I wonder if these memory controls etc. belong in some dedicated message?

Reply:
From glancing at the code briefly, I think this is trying to put a global limit on gRPC stream message sizes vs making it configurable per service, which I think might be placed here: https://github.com/envoyproxy/envoy/blob/45ec050f91407147ed53a999434b09ef77590177/api/envoy/config/core/v3/grpc_service.proto#L288 

If one was doing the latter. 

I think we should either push these limits down to where their used, e.g. in this case per grpc service?, or aggregating them to a context object which is plumbed and widely available.

Doing the latter could add some complexity in cases where we have config for a module in two places, while also making it clearer here are many of the resource knobs.  

WDYT?

Reply:
Note that if you're going to report this in the `GrpcService` proto, then you don't actually need a new field for this at all.  You can set this via the `channel_args` field:

https://github.com/envoyproxy/envoy/blob/45ec050f91407147ed53a999434b09ef77590177/api/envoy/config/core/v3/grpc_service.proto#L267

Reply:
Yeah, maybe we can solve the problem we need internally with existing `ChannelArgs`. @chaoqin-li1123 does this work?

Reply:
This pr is using the ChannelArgs, but we are setting a global limit for all grpc service channels here. maybe we can place the global limit somewhere else?

Reply:
Are you saying a global cummulative limit? I don't think we can easily do that. If it's a global limit that is applied on all gRPC services, I think we can handle that in our internal config translator.

Reply:
I see, it sounds reasonable to implement grpc message size limit in an config translator.